fhd_core/beam_modeling/fhd_struct_init_antenna.pro

```diff
@@ -2,7 +2,7 @@ FUNCTION fhd_struct_init_antenna,obs,beam_model_version=beam_model_version,$
     psf_resolution=psf_resolution,psf_intermediate_res=psf_intermediate_res,$
     psf_image_resolution=psf_image_resolution,timing=timing,$
     psf_dim=psf_dim,psf_max_dim=psf_max_dim,beam_offset_time=beam_offset_time,debug_dim=debug_dim,$
-    inst_tile_ptr=inst_tile_ptr,_Extra=extra
+    inst_tile_ptr=inst_tile_ptr,ra_arr=ra_arr,dec_arr=dec_arr,_Extra=extra
```

Comment:
@isullivan I have a bit of an error with this function that I don't know the most efficient way to handle. I pass ra_arr, dec_arr out. But ra_arr and dec_arr are made with the apply_astrometry function (line 97) with the ignore_refraction flag. I need the ra_arr and dec_arr made with the ignore_refraction flag turned off.